app/models/Office.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+sealed abstract class Office extends Product with Serializable {
+  def value: Option[String]
```

Comment:
Does the value really need to be optional? The parsing code fails if the value is empty so I don't think we can get to this point with a `None`

Reply:
Thanks - I have forgotten to make relax that on the parser! You'll know better than me but I expect there won't be a "CUSOFFDEPEPT" on every message type?

Reply:
No that's correct, but I think we are OK because the `createDeparture` function is only called for IE015 messages in which that tag is mandatory.

Reply:
Good point. I'll remove the Option 👍

Reply:
We're still going to need to keep the readNullable for reading a Departure message from mongo though aren't we so we can accommodate existing departure messages which don't have the new "departureOffice" key?

Reply:
@davidmthornton yes I think you are right, let's keep the Option